src/components/CardModal/CardModal.module.scss

```diff
@@ -0,0 +1,110 @@
+@import '../../styles/variables';
+@import '../../styles/mixins';
+
+.modal {
+  position: fixed;
+  z-index: 100;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  display: flex;
+  overflow: hidden;
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  padding: 1rem;
+  background-color: rgba(0 0 0 / 50%);
+}
+
+.preloader {
+  height: 1.8rem;
+  margin: 1.5rem 3rem 0;
+}
+
+.card {
+  position: relative;
+  width: 400px;
+  animation: show 0.6s ease-in-out;
+  background-color: #fff;
+
+  @include box-shadow;
```

Comment:
if u include this to '.card' then include should be first in styles list

Reply:
fixed